README.md

```diff
@@ -11,8 +11,11 @@ This collection should make it easier to bring Palworld Dedicated Server
 under configuration management on dedicated hardware or an IaaS cloud instance,
 which is a recommended way of running Palworld as it is very resource intensive.
 
-Palworld Dedicated Server will be run as a Docker container on a standard machine,
-managed by Docker Compose.
+Palworld Dedicated Server is configured to run as a Docker container on a standard machine,
+managed by Docker Compose. The Docker container used is
```

Comment:
Alternative word for managed could be:
orchestrated?

Reply:
Changed with the alternative wording. Thank you for the suggestion.